### PR TITLE
Fixed with issue #4 i.e beautify the readme 

### PR DESCRIPTION
**Pull request by pixawna**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,69 +50,70 @@ This is an app that continuously detects the body parts in the frames seen by yo
 
 Added from: [this repo](https://github.com/tensorflow/examples/tree/master/lite/examples)
 
-## Recommedation
+## Recommedation :ghost:
```
Comment (SunitRoy2703) [inline README.md:53]: Can you change ghost emoji to 💬

File: README.md [hunk 2]
```diff
@@ -49,69 +50,70 @@ This is an app that continuously detects the body parts in the frames seen by yo
...
 ## Goals📝
  - [x] Adding all pre-existing example apps to the repo
  - [ ] Designing & Creating other apps using the new [Task API](https://www.tensorflow.org/lite/inference_with_metadata/task_library/overview)
  - [ ] Designing & Creating example apps with the [Interpreter](https://www.tensorflow.org/lite/inference_with_metadata/lite_support), to show the implementation.
  - [ ] Maintaining the Apps
 
- ## Contribute 🤝
- ### Contributions are welcome, checkout [contribution guidelines](./CONTRIBUTING.md)
+ ## :collision:Contribute 🤝
+ #### :wave: Contributions are welcome, checkout [contribution guidelines](./CONTRIBUTING.md) :memo:
```
Comment (SunitRoy2703) [inline README.md:109]: Don't use more than 3 # it makes the text really small

**Review by SunitRoy2703 — CHANGES_REQUESTED** [review]
Awesome @Bhawna1203, let me know when you're done with the changes

File: README.md (hunk 2 above)
Comment (Bhawna1203) [reply README.md:109]: okkk

File: README.md (hunk 1 above)
Comment (Bhawna1203) [reply README.md]: done with this part

**Comment by SunitRoy2703** [comment]
@Bhawna1203 Great work, I'm merging it!

**Pull request merged**